accounts.go

```diff
@@ -342,3 +347,66 @@ func applyUpdateMask(mask *field_mask.FieldMask, msg protoreflect.ProtoMessage,
 	fmutils.Filter(msg, allowedFields)
 	return nil
 }
+
+func (srv *accountsAPI) AuthenticateGoogle(ctx context.Context, in *accountsv1.AuthenticateGoogleRequest) (*accountsv1.AuthenticateGoogleResponse, error) {
+
+	code := in.Code
+
```

Comment:
You forget the validator of the endpoint (see the validator folder) used for error handling and better logs.

Reply:
I check the code with the validator, I have put it in the required parameter. Is it good with that ?